fullscreen.bs

```diff
@@ -254,11 +271,17 @@ these steps:
 
  <li><p>Return <var>promise</var>, and run the remaining steps <a>in parallel</a>.
 
- <li><p>If <var>error</var> is false: Resize <var>pendingDoc</var>'s
- <a>top-level browsing context</a>'s <a>active document</a>'s viewport's dimensions to match the
- dimensions of the screen of the output device. Optionally display a message how the end user can
- revert this.
- <!-- cross-process -->
+ <li>
+  <p>If <var>error</var> is false: Resize <var>pendingDoc</var>'s
+  <a>top-level browsing context</a>'s <a>active document</a>'s viewport's dimensions to match the
+  dimensions of the available screen of the output device, optionally taking into account
+  <var>options</var>'s <code>navigationUI</code> member. Optionally display a message how the end
```

Comment:
You should describe here what the user agent might use the values for. Currently you'd have to infer it from all kinds of non-normative text you could ordinarily ignore.